composer.json

```diff
@@ -16,15 +16,15 @@
     ],
     "require": {
         "php": ">=5.4",
-        "php-http/httplug": "^1.0@dev",
-        "php-http/discovery": "^0.4",
+        "php-http/httplug": "^1.0-beta",
+        "php-http/discovery": "^0.5",
```

Comment:
Isn't it 1.0.0-beta? This might be the reason for failing as travis caches dependencies.